jax/_src/numpy/lax_numpy.py

```diff
@@ -2350,6 +2367,9 @@ def _pad(array, pad_width, mode, constant_values):
   if nd == 0:
     return array
 
+  stat_funcs = {"maximum": jax.numpy.amax, "minimum": jax.numpy.amin,
+                "mean": jax.numpy.mean, "median": jax.numpy.median}
```

Comment:
Same with these: all these aggregations are defined directly in this file so you can drop `jax.numpy`.